### PR TITLE
kubernetes: introduced kubernetes configuration

### PR DESCRIPTION
**Pull request by lpranam**
updated the instructions to setup online in kubernetes

Signed-off-by: Pranam Lashkari <lpranam@collabora.com>
Change-Id: Ic01d94d095eac9dba54a09351eabcbc6ba83c16b

* Resolves: # 
* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: kubernetes/helm/README.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
+In order for Collaborative Editing to function correctly on kubernetes, it is vital to ensure that all users editing the same document end up being served by the same pod. To make this possible we load balance using reverse proxy setup with HAProxy. Using the WOPI protocol, the https headers includes a unique identifier (WOPISrc) for use with this document. Thus load balancing can be done by using WOPISrc – ensuring that all requests that contain the same WOPISrc are sent to the same Collabora Office instance.
+
 ## Helm chart for deploying Collabora Online in Kubernetes cluster
 
 How to test this specific setup:
   1. Install Kubernetes cluster locally - minikube - https://minikube.sigs.k8s.io/docs/
   2. Install helm - https://helm.sh/docs/intro/install/
+  3. install HAProxy Kubernetes Ingress Controller - https://www.haproxy.com/documentation/kubernetes/latest/installation/community/kubernetes/
   3. Prepare the namespace in local kubernetes cluster with this command :
```
Comment (kendy) [inline kubernetes/helm/README.md:11]: I think the rest should be re-numbered? :-)

File: kubernetes/helm/README.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
 # Collabora Online for Kubernetes
 
+In order for Collaborative Editing to function correctly on kubernetes, it is vital to ensure that all users editing the same document end up being served by the same pod. To make this possible we load balance using reverse proxy setup with HAProxy. Using the WOPI protocol, the https headers includes a unique identifier (WOPISrc) for use with this document. Thus load balancing can be done by using WOPISrc – ensuring that all requests that contain the same WOPISrc are sent to the same Collabora Office instance.
```
Comment (timar) [inline kubernetes/helm/README.md:3]: s/includes/include/

File: kubernetes/helm/collabora-online/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: haproxy-configmap
+  namespace: {{ .Values.namespace.collabora }}
+data:
+  timeout-tunnel: 3600s
+  request-set-header: X-HAProxy-loolwsd %[url_param(WOPISrc)]
```
Comment (timar) [inline kubernetes/helm/collabora-online/templates/configmap.yaml:8]: We've just got a feedback from @juliushaertl that this HAProxy setup did not work well with full screen presentation, and he suggested a different setting. @lpranam could you please verify it?
* request-set-header is not needed.
* load-balance: balance url_param WOPISrc check_post

Comment (lpranam) [reply kubernetes/helm/collabora-online/templates/configmap.yaml:8]: yes, I just checked presentation is not loading.

**Review by timar — APPROVED** [review]

**Pull request merged**